### PR TITLE
revamp documentation

### PR DESCRIPTION
**Pull request by itaysk**
rearranged all the documentation into the /docs folder and simplified the docs site structure.

the idea is that the readme will contain just a simple intro to the project, and the full docs will be separate (like in Trivy/Starboard).

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -5,117 +5,34 @@
...
-`--webhook-content-type` | If present, will set the Content-Type HTTP header to match the provided template | `--webhook-content-type application/json`
-
-A popular webhook server that can be used with Tracee is [falcosidekick](https://github.com/falcosecurity/falcosidekick), which can send detection events into other systems of your choosing (for example Slack, Teams, Datadog, Prometheus, Email, Elasticsearch, PagerDuty, and many more). For more information on falcosidekick see [here](https://github.com/falcosecurity/falcosidekick/blob/master/config_example.yaml).
-
-### Setup options
-
-Tracee is leveraging Linux's eBPF technology, which is kernel and version sensitive, and therefore needs to be specifically compiled for your hosts.
+> Note: You may need to change the volume mounts for the kernel headers based on your setup. See [Linux Headers](https://aquasecurity.github.io/tracee/install/headers.md) section for more info..
```
Comment (grantseltzer) [inline Readme.md:27]: extra `.`

File: Readme.md [hunk 2]
```diff
@@ -5,117 +5,34 @@
...
 
-Tracee is composed of the following sub-projects:
-- [Tracee-eBPF](tracee-ebpf) - Linux Tracing and Forensics using eBPF
-- [Tracee-Rules](tracee-rules) - Runtime Security Detection Engine
-- [libbpgo](libbpfgo) - Go library for eBPF programming using Linux's [libbpf](https://github.com/libbpf/libbpf)
-
-## Getting started
-
-### Prerequisites
+The full documentation of Tracee is available at [https://aquasecurity.github.io/tracee](https://aquasecurity.github.io/tracee).
```
Comment (grantseltzer) [inline Readme.md:14]: Do I understand correctly that this link won't work until after this merges? Because it's broken right now.

Comment (itaysk) [reply Readme.md:14]: yes you understand correctly

**Review by grantseltzer — APPROVED** [review]
lgtm!

**Comment by simar7** [comment]
Do we have to configure GitHub Pages in a certain way for this to work? Usually it takes a few hours for them to go live but it's been several at the time of writing this and https://aquasecurity.github.io/tracee is still a 404 for me.

**Comment by itaysk** [comment]
It will be there after we release (git push is the trigger)

**Pull request merged**